t/t-track.sh

```diff
@@ -595,3 +595,28 @@ begin_test "track (system gitattributes)"
   grep "*.dat" track.log
 )
 end_test
+
+begin_test "track: escaped pattern in .gitattributes"
+(
+  set -e
+
+  reponame="track-escaped"
+  git init "$reponame"
+  cd "$reponame"
+
+  filename="file with spaces.#"
+
+  echo "I need escaping" > "$filename"
+
+  [ "Tracking \"$filename\"" = "$(git lfs track "$filename")" ]
+  [ "\"$filename\" already supported" = "$(git lfs track "$filename")" ]
+  
+  #changing flags should track the file again
+  [ "Tracking \"$filename\"" = "$(git lfs track -l "$filename")" ]
+
+  if (( 1 != $(wc -l < .gitattributes) )); then
```

Comment:
Nit; we prefer the `[`-style `test` invocation, like:

```bash
if [ 1 -ne "$(wc -l .gitattributes | awk '{ print $1 }')" ]; then
  # ..
fi
```

A couple of other improvements that are written above:

- The `<` redirection into `wc(1)` is unnecessary, since `wc` accepts file(s) as arguments.
- `-ne` is preferred over `!=`.
- `awk` to print only the numeric part of `wc`'s output.

Reply:
sure, whatever you say. I'm not very strong at bash scripting, but I know enough to just follow advises of knowledgeable people.